docs/en/reference/index.rst

```diff
@@ -1,85 +1,5 @@
 Introduction
 ============
 
-The Doctrine Reflection project is a simple library used by the various Doctrine projects which adds some additional
-functionality on top of the reflection functionality that comes with PHP. It allows you to get the reflection information
-about classes, methods and properties statically.
-
-Installation
-============
-
-The library can easily be installed with composer.
-
-.. code-block:: sh
-
-    $ composer require doctrine/reflection
-
-Setup
-=====
-
-.. code-block:: php
-
-    use Doctrine\Common\Reflection\Psr0FindFile;
-    use Doctrine\Common\Reflection\StaticReflectionParser;
-    use App\Model\User;
-
-    $finder = new Psr0FindFile(['App' => [
-        '/path/to/project/src/App'
-    ]]);
-
-    $staticReflectionParser = new StaticReflectionParser(User::class, $finder);
-
-Usage
-=====
-
-.. code-block:: php
-
-    echo $staticReflectionParser->getClassName();
-    echo $staticReflectionParser->getNamespaceName();
-
-StaticReflectionClass
-=====================
-
-.. code-block:: php
-
-    $staticReflectionClass = $staticReflectionParser->getReflectionClass();
-
-    echo $staticReflectionClass->getName();
-
-    echo $staticReflectionClass->getDocComment();
-
-    echo $staticReflectionClass->getNamespaceName();
-
-    print_r($staticReflectionClass->getUseStatements());
-
-StaticReflectionMethod
-======================
-
-.. code-block:: php
-
-    $staticReflectionMethod = $staticReflectionParser->getReflectionMethod('getSomething');
-
-    echo $staticReflectionMethod->getName();
-
-    echo $staticReflectionMethod->getDeclaringClass();
-
-    echo $staticReflectionMethod->getNamespaceName();
-
-    echo $staticReflectionMethod->getDocComment();
-
-    print_r($staticReflectionMethod->getUseStatements());
-
-StaticReflectionProperty
-========================
-
-.. code-block:: php
-
-    $staticReflectionProperty = $staticReflectionParser->getReflectionProperty('something');
-
-    echo $staticReflectionProperty->getName();
-
-    echo $staticReflectionProperty->getDeclaringClass();
-
-    echo $staticReflectionProperty->getDocComment();
-
-    print_r($staticReflectionProperty->getUseStatements());
+The main functionality of this package has been deprecated, you should
+not use it.
```

Comment:
Tell me what you think about this: does Doctrine usually document deprecated functionality? IMO it's great not to encourage people to build more on it.

Reply:
Wording is strange. So are we deprecating the package, or not? If we are, wording should say so. If we aren't, why is this saying to not use the package if there is still some use?

Reply:
Yeah… if the main functionality of the package is deprecated, then what should it be used for and what's its new main functionality? Deprecating the whole thing and moving the relevant pieces elsewhere might be a more proper decision.